projects/go-lib/src/lib/components/go-select/go-select.component.html

```diff
@@ -1,4 +1,12 @@
+<label
+  class="go-form__label"
+  [attr.for]="id"
+  [ngClass]="{'go-form__label--dark': theme === 'dark'}">
+  {{ label }}
+</label>
+
 <ng-select
+  [id]="id"
```

Comment:
It looks like ng-select has a `[labelForId]` input that we could use to better associate the label to the input.

https://github.com/ng-select/ng-select/blob/master/src/ng-select/lib/ng-select.component.ts#L106

Reply:
Good catch. Switched it to use [labelForId]